cellprofiler/gui/figure.py

```diff
@@ -650,10 +650,14 @@ def on_mouse_move(self, evt):
             y0 = evt.ydata
             y1 = evt.ydata
         else:
-            x0 = min(self.mouse_down[0], evt.xdata)
-            x1 = max(self.mouse_down[0], evt.xdata)
-            y0 = min(self.mouse_down[1], evt.ydata)
-            y1 = max(self.mouse_down[1], evt.ydata)
+            x0, x1 = self.mouse_down[0], self.mouse_down[0]
```

Comment:
Another thought - could this be done like so?

```
x = self.mouse_down[0]
x0 = min(x, evt.xdata or x)
x1 = max(x, evt.xdata or x)
```
The `or` would then default to `x` if it was `None`

Reply:
You may be right there. I expect there are a few ways to achieve this, so I wasn't sure which would be the best solution.